app/views/imports/_cases.html.erb

```diff
@@ -1,20 +1,37 @@
-<h3><i class="fa fa-download fa-2x" aria-hidden="true"></i> 1. Download example Case CSV file</h3>
-<%= link_to "/casa_cases.csv", format: :csv, class: "btn btn-xs btn-success" do %>
-  <i class="fa fa-download"></i> Download example Case csv
-<% end %>
-<br>
-<br>
-<h3><i class="fa fa-file-text-o fa-2x" aria-hidden="true"></i> 2. Upload your CSV file</h3>
-<%= form_with(url: imports_path, local: :true) do |f| %>
+<div class="mb-3">
+  <h4>
+    1.
+    <%= link_to "/casa_cases.csv", format: :csv, download: "casa_cases.csv" do %>
+      Download and reference example Case CSV file
+      <i class="fa fa-file-download"></i>
+    <% end %>
+  </h4>
+</div>
+
+<div class="mb-3">
+  <h4>
+    2. Upload your CSV file
+    <i class="fa fa-file-csv" aria-hidden="true"></i>
+  </h4>
+  <%= form_with(url: imports_path, local: :true) do |f| %>
   <%= f.hidden_field :import_type, value: "casa_case" %>
   <ul>
     <li>Click the choose file button and navigate to the saved file and select it.</li>
     <li><strong>Do not</strong> change any of the values in the first line of the example csv file.</li>
     <li>Then click the "Import Cases CSV" button to import your cases.</li>
   </ul>
-  <%= f.file_field :file, accept: 'text/csv', class: 'form-control-file', style: "margin: auto;" %>
-  <br>
-  <%= button_tag :class => "btn btn-md btn-success" do %>
-    <i class="fa fa-upload"></i> Import Cases CSV
-  <% end %>
+  <%= f.file_field :file, id: 'case-file', accept: 'text/csv', class: 'form-control-file', style: "margin: auto;" %>
+</div>
+
+<%= button_tag id: "case-import-button", :class => "btn btn-md btn-success pull-right", 
```

Comment:
This is more like a personal preference so it is not a blocker, I think we could change the `:class => "btn btn-md btn-success pull-right"` to use the new format `class: "btn btn-md btn-success pull-right"`